<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>792</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="opaqueResize">
    <bool>true</bool>
   </property>
   <property name="handleWidth">
    <number>2</number>
   </property>
   <property name="childrenCollapsible">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(47, 47, 47);
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="media_pause">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;暂停&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/4.png</normaloff>:/image/4.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="9">
        <widget class="QSlider" name="video_position">
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
border: 1px solid #4A708B;
background: #C0C0C0;
height: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
}
QSlider::sub-page:horizontal {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #B1B1B1, stop:1 #c4c4c4);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #5DCCFF, stop: 1 #1874CD);
border: 1px solid #4A708B;
height: 10px;
border-radius: 2px;
}

QSlider::add-page:horizontal {
background: #575757;
border: 0px solid #777;
height: 10px;
border-radius: 2px;
}

QSlider::handle:horizontal 
{
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
    stop:0.6 #45ADED, stop:0.778409 rgba(255, 255, 255, 255));

    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #2A8BDA, 
    stop:0.778409 rgba(255, 255, 255, 255));

    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}

QSlider::sub-page:horizontal:disabled {
background: #00009C;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="11">
        <widget class="QLabel" name="label_video">
         <property name="styleSheet">
          <string notr="true">QLabel{
	background-color: rgb(47, 47, 47);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="media_play">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/12.png</normaloff>:/image/12.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="6" colspan="2">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="label_time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>00:00:00/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="8" colspan="2">
        <widget class="QSlider" name="video_sound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_total">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>508</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="media_right_play">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;前进15s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="media_stop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;停止&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/11.png</normaloff>:/image/11.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="media_left_play">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;后退15s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/9.png</normaloff>:/image/9.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
	min-width:85px;
	min-height:27px;
	font: 11pt &quot;Agency FB&quot;;
	background-color: rgb(59, 59, 59);
	color: rgb(255, 255, 255);
}

QTabBar::tab:selected{
	min-width:85px;
	min-height:27px;
	font: 11pt &quot;Agency FB&quot;;
	background-color: rgb(59, 59, 59);
	color: rgb(255, 255, 255);
}

QTabWidget::pane
{
	border-width: 0px;
	border-radius: 0px;
	background-color: rgb(59, 59, 59);
}​</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>计算机</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTreeView" name="treeView">
        <property name="styleSheet">
         <string notr="true">QTreeView{
	background-color: rgb(59, 59, 59);
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>链接播放</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">	background-color: rgb(59, 59, 59);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>历史播放</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">QListWidget{
	background-color: rgb(59, 59, 59);
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
